### PR TITLE
Ewc 99: remove send to debugger & viewer

### PR DESCRIPTION
**Pull request by ilaryionava**

### CODE REVIEW

**Comment by ekrylovich** [comment]
Why did you remain all this code in comments??

File: src/public/js/build/build-interface.js [hunk 1]
```diff
@@ -1629,6 +1599,7 @@ function findInLayoutDictionary(id) {
 }
 
 function updateDebugViewerRoute(event) {
```
Comment (epankou) [inline src/public/js/build/build-interface.js]: do we need that function at all?

File: src/server/controllers/api/buildController.js [hunk 2]
```diff
@@ -139,17 +139,17 @@ const outputData = function(params, callback) {
       fogInstanceId: instance.fogInstanceId,
       rootHostAccess: instance.rootHostAccess,
       logSize: instance.logSize,
-      viewerEnabled: instance.viewerEnabled,
-      debugEnabled: instance.debugEnabled,
+      viewerEnabled: 0,
```
Comment (epankou) [inline src/server/controllers/api/buildController.js]: is it still required

File: src/public/js/build/build-interface.js [hunk 3]
```diff
@@ -1320,15 +1299,6 @@ function updateElementInstance(name) {
         deleteRoute(previousFabric, instanceId, previousFabric, 'debug');
```
Comment (elukashick) [inline src/public/js/build/build-interface.js]: May it also be removed?

File: src/public/js/build/build-interface.js (hunk 1 above)
Comment (ilaryionava) [reply src/public/js/build/build-interface.js]: deleted

**Pull request merged**